usaspending_api/broker/helpers/find_related_legal_entities.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models import Count
+from usaspending_api.awards.models import TransactionNormalized
+
+
+def find_related_legal_entities(transactions):
+    related_le_ids = [result[0] for result in transactions.values_list('recipient_id')]
```

Comment:
Another potential benefit of this method is that it will embed the `related_le_ids` query in the `tn_count` rather than embedding the raw values.  Sometimes one query is better than two.  Sometimes it's not.

Reply:
Ah yes I totally forgot about `flat`.